qiskit/circuit/quantumcircuit.py

```diff
@@ -1575,11 +1575,11 @@ def qasm(
         filename: Optional[str] = None,
         encoding: Optional[str] = None,
     ) -> Optional[str]:
-        """Return OpenQASM string.
+        """Return OpenQASM2 string.
```

Comment:
As long as we are editing the line, it makes sense to improve it as well. Maybe

"Return a string representing the `QuantumCircuit` as an OpenQASM 2.0 program."

"Convert the `QuantumCircuit` to an OpenQASM 2.0 program and return the resulting string."

Reply:
good point. Addressed in f7f5756ae27b59cd20390357dc5d6559c5adc27c